.github/workflows/main.yml

```
@@ -1094,9 +1094,9 @@ jobs:
           $metadata = Get-Content -Raw "C:\Card-Game-Simulator.git\msstore\metadata.json" 
           msstore submission updateMetadata 9N96N5S4W3J0 $metadata
       - name: Publish to Store
+        if: false
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)